aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -324,7 +338,7 @@ impl AggregatorData {
             limit,
             history: None,
         };
-        self.aggregators.insert(id, aggregator);
+        self.aggregators.insert(id.clone(), aggregator);
```

Comment:
interesting, it didn't complain before?

Reply:
Before ids were Copy, but if using state key it is not

Reply:
Reverted to the old flow